src/components/filter/Filter.tsx

```diff
@@ -29,21 +30,80 @@ type Props = {
 export const Filter = ({
   filterTypes,
   initialFilters,
+  initialStartDate,
   isOverlay = false,
   queryVariables,
   setQueryVariables,
   withSearch = false
 }: Props) => {
-  const [filters, setFilters] = useState<FilterProps>(queryVariables);
+  const [filters, setFilters] = useState<FilterProps>(
+    initialStartDate
+      ? Object.fromEntries(
+          Object.entries(queryVariables).filter(
+            ([key, value]) => !(key === 'start_date' && value === initialStartDate)
+          )
+        )
+      : queryVariables
+  );
   const [isCollapsed, setIsCollapsed] = useState(true);
   const [filterCount, setFilterCount] = useState(0);
 
   useEffect(() => {
     if (!isOverlay) {
-      setQueryVariables((prev) => ({ search: prev.search || '', ...filters }));
+      setQueryVariables((prev) => {
+        const newFilters = { ...filters };
+
+        if (newFilters.start_date === initialStartDate) {
+          delete newFilters.start_date;
+          return {
+            ...prev,
+            search: prev.search || '',
+            start_date: initialStartDate,
+            ...newFilters
+          };
+        }
+
+        if (!newFilters.start_date) {
+          return {
+            ...prev,
+            search: prev.search || '',
+            start_date: initialStartDate,
+            ...newFilters
+          };
+        }
+
+        return { ...prev, search: prev.search || '', ...newFilters };
+      });
     }
   }, [filters]);
 
+  const resetFilters = () => {
+    if (!isOverlay) {
```

Comment:
how come the difference between overlay and not overlay?
why do we have separate logics here?

Reply:
Because the Filter component is used in two different ways. One is the Overlay-Filter and the other is the SUE-Filter. We have the same logic in the return section.